test-app/src/test/kotlin/uk/gov/justice/digital/hmpps/testapp/integration/customizer/AuthenticateHttpRequestsCustomizerTest.kt

```diff
@@ -0,0 +1,81 @@
+package uk.gov.justice.digital.hmpps.testapp.integration.customizer
+
+import org.junit.jupiter.api.Test
+import org.springframework.boot.test.context.TestConfiguration
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Import
+import uk.gov.justice.digital.hmpps.testapp.integration.IntegrationTestBase
+import uk.gov.justice.hmpps.kotlin.customize.ResourceServerConfigurationCustomizer
+
+@Import(AuthenticateHttpRequestsCustomizerTest.CustomizerConfiguration::class)
+class AuthenticateHttpRequestsCustomizerTest : IntegrationTestBase() {
+
+  @TestConfiguration
+  class CustomizerConfiguration {
+    @Bean
+    fun configurationCustomizer() = ResourceServerConfigurationCustomizer.build {
+      authorizeHttpRequests {
+        authorize("/info", hasRole("INFO"))
+        authorize(anyRequest, hasRole("ANY_REQUEST"))
+      }
+    }
+  }
```

Comment:
The main change is to allow the default `authorizeHttpRequests` DSL to be replaced.